### PR TITLE
Returns the Vox, who now breath Phoron

### PR DESCRIPTION
**Pull request by Anewbe**
To clarify, they're for admin/event use only as of now.
Also swapped the emergency oxygen tanks to a new path, shouldn't do much.

### CODE REVIEW

**Comment by Techhead0** [comment]
RIP Polaris maps.

**Comment by Anewbe** [comment]
Damn tool done eat muh maps!

File: code/ZAS/Phoron.dm [hunk 1]
```diff
@@ -40,18 +40,28 @@ var/image/contamination_overlay = image('icons/effects/contamination.dmi')
 
 obj/var/contaminated = 0
 
+obj/var/phoronproof = 0
```
Comment (comma) [inline code/ZAS/Phoron.dm]: Why add this instead of using PHORONGUARD flag?

Comment (Anewbe) [inline code/ZAS/Phoron.dm]: For whatever reason the flag wasn't working. Tried that first.

**Comment by FlattestGuitar** [comment]
Yaya!

Kikikiki.

**Comment by Spacemanspark** [comment]
Voxxy will take all shinies on Polaris too?
Is voxxy dreams come true! Skreeeeeee

**Comment by SpadesNeil** [comment]
KIKIKIKIKI.

**Comment by SpadesNeil** [comment]
**# ALL ABOARD THE HYPE TRAIN. DESTINATION: YOUR SIDES.**

**Comment by Spacemanspark** [comment]
SKREKS WILL NEVER SEE IT COMINGS.

**Comment by Zuhayr** [comment]
You need to be aware that @comma links me to this PR every time he wants to make me upset.

**Comment by Anewbe** [comment]
I whined for a full year before they let me make this monstrosity.

**Pull request merged**